parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueReader.java

```diff
@@ -28,5 +28,14 @@ public interface ParquetValueReader<T> {
 
   List<TripleIterator<?>> columns();
 
+  /**
+   * @deprecated since 1.6.0, will be removed in 1.7.0; use setPageSource(PageReadStore) instead.
```

Comment:
```suggestion
   * @deprecated since 1.8.0, will be removed in 1.9.0 or 2.0.0; use setPageSource(PageReadStore) instead.
```

Reply:
In my change, I put down 1.9.0. If there is no 1.9.0, and the methods are removed in 2.0.0 instead, I don't think that would be a problem.